src/lib/config.js

```diff
@@ -14,6 +14,7 @@ const defaultConfig = {
 		someConsentGiven: 30,
 		noConsentGiven: 30,
 	},
+	geoIPVendor: 'http://cmp.digitru.st/geoip-demo.html',
```

Comment:
Let's use a different path than /geoip-demo.html

Reply:
Let me know what the new URL is and I'll be happy to test it out and update it :)

Reply:
Let's go with `http://cmp.digitru.st/geoip.json` for now